src/main.cpp

```diff
@@ -2784,6 +2786,42 @@ void static UpdateTip(CBlockIndex *pindexNew) {
       Checkpoints::GuessVerificationProgress(chainParams.Checkpoints(), chainActive.Tip()), pcoinsTip->DynamicMemoryUsage() * (1.0 / (1<<20)), pcoinsTip->GetCacheSize());
 
     cvBlockChange.notify_all();
+    
+    
+
+    // MVHF-BU
+    const int MVHF_FORK_BLOCK = 50000;
+
+    //// 	Auto Backup At Block 		////
+    // Test autobackupwalletpath argument
+    if (!fAutoBackupDone)
+    {
+    	std::string strWalletBackupFile = GetArg("-autobackupwalletpath", "");
+
+		int BackupBlock = GetArg("-autobackupblock", MVHF_FORK_BLOCK - 1);
```

Comment:
so you're doing a backup at the end of block 113 (speaking in terms of the current test), i.e. the semantic is that the backup is done before the specified block number? (this is all assuming we stick to the convention of starting block numbering with 0 - the genesis block)

if so, we may need to adapt MVHF-BU-SW-REQ-10-2 which mentions "AFTER" , or we change the condition below from >= to > , or we change this statement to eliminate the " - 1" ?

What is your opinion on the best course of action?